hackathon_site/hardware/api_filters.py

```diff
@@ -63,4 +71,4 @@ class OrderFilter(filters.FilterSet):
         help_text="Comma separated list of team IDs",
     )
     team_code = filters.CharFilter(field_name="team__team_code")
-    status = filters.CharFilter(field_name="status")
+    status = filters.CharFilter(field_name="status")
```

Comment:
nitpick: could you add a newline at the end of the file (same goes for all the files you changed)

Reply:
ok sure